jx-requirements.yml

```diff
@@ -31,6 +31,4 @@ spec:
       production: false
   repository: bucketrepo
   secretStorage: vault
-  vault:
```

Comment:
No need to set it for internal vault created by jx. It uses the right default value.